src/core/energy_storage/electric_storage.jl

```diff
@@ -283,13 +306,15 @@ struct ElectricStorage <: AbstractElectricStorage
         # copy the replace_costs in case we need to change them
         replace_cost_per_kw = s.replace_cost_per_kw 
         replace_cost_per_kwh = s.replace_cost_per_kwh
+        replace_cost_constant = s.replace_cost_constant
         if s.model_degradation
             if haskey(d, :replace_cost_per_kw) && d[:replace_cost_per_kw] != 0.0 || 
                 haskey(d, :replace_cost_per_kwh) && d[:replace_cost_per_kwh] != 0.0
```

Comment:
should probably add a check for the replace_cost_constant here too, right? e.g.: 
```
||  haskey(d, :replace_cost_constant) && d[:replace_cost_constant] != 0.0
```

Reply:
Added, thanks!